webapp/src/app/components/sidenav/sidenav.component.html

```diff
@@ -1,5 +1,5 @@
 <div fxLayout="column" class="nav-container" *ngIf="tilemap">
-	<mat-tab-group (selectedTabChange)="tabChanged($event)" [selectedIndex]="activeTab">
+	<mat-tab-group (selectedTabChange)="tabChanged($event)" [(selectedIndex)]="activeTab">
```

Comment:
Is the `selectedIndex` event really necessary? In this case activeTab is set twice, once from `(selectedTabChange)` and once from `(selectedIndex)`

Reply:
The binding seems to be unnecessary. Removed in 6ead0628ebbbc7ece8c82c7d0f47a99749600658.